data-prepper-plugins/opensearch-source/build.gradle

```diff
@@ -0,0 +1,28 @@
+plugins {
```

Comment:
I see that this is not added to the `settings.gradle`. Is this file missing from the commit?

-----

Also, you do not need lines 1-10. This is all configured by the root project. Please delete these lines.

Reply:
Thanks @dlvenable for the review. The changes are incorporated.